src/Management/src/OpenTelemetryBase/Steeltoe.Management.OpenTelemetryBase.csproj

```diff
@@ -15,11 +13,10 @@
 
   <ItemGroup>
     <PackageReference Include="OpenTelemetry" Version="$(OpenTelemetryVersion)" />
-    <PackageReference Include="OpenTelemetry.Exporter.Zipkin" Version="$(OpenTelemetryVersion)" />
```

Comment:
Dropped this to avoid forcing a dependency on Zipkin package... Autoconfig can still bootstrap Zipkin, but any other exporter would require tracing to be manually setup.

Side affect is UseShortTraceId only affects log processor now

Reply:
It seems to make sense, but I don't have a full grasp of the implications.  Seems like we can document the manual setup?  Do you agree @TimHess?   @hananiel Do you have any thoughts on this?

Reply:
TracingBase can now auto-wire 3 different OTel exporters, without forcing the nuget dependency